DEVELOPERS.md

```diff
@@ -30,6 +30,15 @@ for more info. Example usage can be found [here](http://www.doxygen.nl/manual/do
 static int findMinMax(RzList *maps, ut64 *min, ut64 *max, int skip, int width);
 ```
 
+In order to improve the documentation and help newcomers, documenting code is mandatory.
+
+You should add or update the documentation of:
+- code written by you.
+- existing Rizin code you changed.
```

Comment:
Maybe add an exception for trivial changes, since often those could be done without understanding what the modified function does.

Reply:
Adding an exception besides the example?
Or might it be enough to add another example here:

https://github.com/rizinorg/rizin/blob/0db022224578652374cc9127131f9ca888f21fc5/DEVELOPERS.md?plain=1#L39-L40
Something like:
"Changes are trivial and don't need documentation."
"Getter and setter are trivial and don't need documentation."

Reply:
Never mind. The exception clause is more inclusive.